### DISCUSSION TITLE
non viene caricata interfacccia Lovelace dopo login

### DISCUSSION THREAD

**Discussion by LORE-64** [indomus/forum#634] [HUB: Home Assistant]
Salve a tutti ho un problema
Dopo il login da interfaccia web sia internamente alla Lan sia esternamente alla Lan, non viene caricata la pagina ma rimane perennemente in attesa caricamento.
Installato Home assistance Core su rasberry con SD card

Cosa posso fare?

**Comment by snakuzzo** [comment]
Controllare i log di HA

**Comment by LORE-64** [comment]
indi devo connettermi direttamente con tastiera e e video al rasberry?

**Reply by snakuzzo** [reply]
Per favore non rispondere in nuovi thread ma fai reply sotto l'ultimo messaggio a cui vuoi rispondere, se si sta parlando della stessa cosa.

Non so che tipo di installazione tu abbia fatto, ma immagion HassOS a questo punto. E la risposta è "sì"
